backend-modules/blockfrost/src/main/java/com/bloxbean/cardano/client/backend/blockfrost/service/BFAssetService.java

```diff
@@ -119,4 +120,23 @@ public Result<List<PolicyAsset>> getPolicyAssets(String policyId, int count, int
     public Result<List<PolicyAsset>> getPolicyAssets(String policyId, int count, int page) throws ApiException {
         return getPolicyAssets(policyId, count, page, OrderEnum.asc);
     }
+
+    @Override
+    public Result<List<AssetTransactionContent>> getTransactions(String unit, int count, int page, OrderEnum order) throws ApiException {
+        if(order == null)
+            order = OrderEnum.asc;
+
+        Call<List<AssetTransactionContent>> call = assetsApi.getTransactions(getProjectId(), unit, count, page, order.toString());
+
+        try {
+            Response<List<AssetTransactionContent>> response = call.execute();
+            if (response.isSuccessful())
```

Comment:
I think we can replace these two lines with processResponse method call

```
processResponse(response) 
```

Reply:
Have adjusted